spec/draft/profile-spec.md

```diff
@@ -183,7 +183,7 @@ ModelDefinition :
 
 ModelSpecification: 
 
-- ModelDefinition
+- ModelDefinition RequiredValue?
```

Comment:
Why is this on `ModelSpecification` instead of `FieldSpecification` ? The implication is that it now can be used in list model definitions:

```
model MyList [ ListItem! ]
```

Is this intentional?

Reply:
It is intentional. Second place affected is result:

```
usecase Examples {
  result string!
}
```

As we have values optional (nullable) by default, there was no way to express that the result or item of the array can't be null.

At least according to specification it wasn't possible because implementation actually allows it, so this is just fixing it in specification.

eg most recent use: https://github.com/superfaceai/station/pull/328/files#diff-1214f8f382177fe4d668a4a0dccef94666c9df069b550a9f3655a8b0ba901b0fR68

Reply:
> because implementation actually allows it

@freaz so this is already implemented in the parser and validator?

Reply:
Precisely. It is how it works right now, and this just aligns specs.